packages/mobile/e2e/src/usecases/PINChange.js

```diff
@@ -19,13 +18,27 @@ export default ChangePIN = () => {
       .withTimeout(5000)
     await element(by.id('ChangePIN')).tap()
     // Existing PIN is needed first
+    await sleep(500)
```

Comment:
Why do we need all these sleeps? Can we do `waitFor`s instead or something like that? Plain `sleep`s look like they could cause flakiness :(

Reply:
This is because the new pin entry disabled detox syncing to type in faster; however, I agree a wait for seems more ideal here. I'll update these sleeps to `waitFor`s.

Reply:
Tried with some `waitFor` and adding some new testID's and it was actually slower and flakier. As this all occurring in a section where [device synchronization is disabled](https://github.com/wix/detox/blob/master/docs/APIRef.DeviceObjectAPI.md#devicedisablesynchronization) sleeps should be ok.